here-naksha-lib-psql/src/commonMain/kotlin/naksha/psql/executors/PgReader.kt

```diff
@@ -55,10 +55,12 @@ class PgReader(
         val query = ReadQueryBuilder().build(request)
         val connection = session.usePgConnection()
         val cursor = connection.execute(query.rawSql)
-        cursor.fetch()
-        val rawTupleNumber = cursor.column(tuple_number)
-        if (rawTupleNumber is ByteArray) {
-            val tupleNumberBytes = TupleNumberByteArray(storage, rawTupleNumber)
+        var allBytes: ByteArray = byteArrayOf()
+        while(cursor.next()){
+            val bytes: ByteArray = cursor.column(tuple_number) as ByteArray
+            allBytes += bytes
+        }
```

Comment:
Please add `cursor.close()` after we've done reading from it (I think it wasn't there before but I just noticed)